Sprint-1/exercises/decimal.js

```diff
@@ -8,6 +8,8 @@ const num = 56.5678;
 
 // Log your variables to the console to check your answers
 const wholeNumberPart = Math.floor(num);
-const decimalPart = +(num - wholeNumberPart).toFixed(2); 
+const decimalPart = +(num - wholeNumberPart).toFixed(4); 
+const rounded = Math.round(num);
```

Comment:
Great solution, although the variable name you used doesn't match the one you were instructed to use.